### PR TITLE
10.0 pos small fixes

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Review by yelizariev — CHANGES_REQUESTED** [review]
README was used as description for apps store. If we update README, we need new html description. Try to make something from our templates or make a request for a description

**Comment by yelizariev** [comment]
This branch has conflicts that must be resolved

File: pos_keyboard/README.rst [hunk 1]
```diff
@@ -8,8 +8,6 @@
 
 The module allows to apply the usual keyboard (not the virtual one) in the Point of Sale.
 
-Usage instructions: `<doc/index.rst>`_
```
Comment (yelizariev) [inline pos_keyboard/README.rst]: This commit fix updates of another commit of the same PR. So, you need to merge such commits

File: pos_pin/README.rst [hunk 2]
```diff
@@ -15,7 +15,7 @@ Credits
 
 Contributors
 ------------
-* `Krotov Stanislav <https://github.com/ufaks>`__
+* `Krotov Stanislav <https://www.it-projects.info/team/ufaks>`__
 * `Kolushov Alexandr <https://it-projects.info/team/KolushovAlexandr>`__
```
Comment (yelizariev) [inline pos_pin/README.rst:19]: Different url style (with www and without www)

File: pos_pin/static/src/js/pos.js [hunk 3]
```diff
@@ -54,6 +54,7 @@ odoo.define('pos_pin.pos', function (require) {
                 if (options.security && cashier !== options.current_user && cashier.pos_security_pin) {
                     return self.ask_password(cashier.pos_security_pin, options.arguments);
                 }
+                return cashier;
```
Comment (yelizariev) [inline pos_pin/static/src/js/pos.js:57]: Changelog is needed I think

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Review by yelizariev — APPROVED** [review]

**Pull request merged**